app/code/Magento/Catalog/Ui/DataProvider/Product/Form/Modifier/Eav.php

```diff
@@ -539,6 +539,10 @@ private function prepareGroupSearchCriteria()
      */
     private function getAttributeSetId()
     {
+        if((int)$this->request->getParam('set', 0)){
+            return (int)$this->request->getParam('set', 0);
```

Comment:
I think you can avoid passing the default value `0` in the second call to `getParam()` because at that point you are sure the param exists.
What do you think about it?

Reply:
yes you are right . basically here Idea was to get attribute set value from parameter if that exist.I am making changes in the file